spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/SpatialConvolution.scala

```diff
@@ -44,7 +45,9 @@ class SpatialConvolution[T: ClassTag](
   val padH: Int = 0, // The additional zeros added per height to the input planes.
   val nGroup: Int = 1, // Kernel group number
   val propagateBack: Boolean = true, // propagate gradient back
-  private var initMethod: InitializationMethod = Default
+  private var initMethod: InitializationMethod = Default,
+  val wRegularizer: Regularizer[T] = null,
```

Comment:
The constructor list is too long. Can we move the new constructor parameter out?